src/javascript/expression.rs

```diff
@@ -326,6 +334,43 @@ impl<'module> Generator<'module> {
             .append(" ")
             .append(right))
     }
+
+    fn todo<'a>(&mut self, message: &'a Option<String>, location: &'a SrcSpan) -> Document<'a> {
+        self.tail_position = false;
+        let gleam_error = "todo";
+        let message = message
+            .as_ref()
+            .map(|s| s.as_str())
+            .unwrap_or_else(|| "This has not yet been implemented");
+        let module_name = Document::String(self.module_name.join("_"));
+        let line = location.start;
```

Comment:
This isn't a line number, it's a byte offset. We'll need to convert it like this

https://github.com/gleam-lang/gleam/blob/5719757fd20e1324f492fcba466e5aa9e5d2f549/src/erl.rs#L1360